src/plugins/home/server/services/sample_data/data_sets/otel/logs_field_mappings.ts

```diff
@@ -0,0 +1,343 @@
+                ignore_above: 256,
+              },
+            },
+          },
+          'process@executable@path': {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          'process@owner': {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          'process@pid': {
+            type: 'long',
+          },
+          'process@runtime@description': {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          'process@runtime@name': {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          'process@runtime@version': {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          'service@name': {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          'service@version': {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          'telemetry@auto@version': {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          'telemetry@sdk@language': {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          'telemetry@sdk@name': {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+          'telemetry@sdk@version': {
+            type: 'text',
+            fields: {
+              keyword: {
+                type: 'keyword',
+                ignore_above: 256,
+              },
+            },
+          },
+        },
+      },
+    },
+  },
+  schemaUrl: {
+    type: 'text',
+    fields: {
+      keyword: {
+        type: 'keyword',
+        ignore_above: 256,
+      },
+    },
+  },
+  serviceName: {
+    type: 'keyword',
+    fields: {
+      keyword: {
+        type: 'keyword',
+        ignore_above: 256,
```

Comment:
would be better to remove the `.keyword` field given it is already keyword type

Reply:
Sure, will mark this as discussed already with vamsi that we will enhance sample data experience including things like this.